metadata.yaml

```diff
@@ -39,3 +39,10 @@ resources:
     type: oci-image
     description: Container image for Prometheus
     upstream-source: docker.io/ubuntu/prometheus@sha256:03936a740851b8786c328e08803a70feaa97731714a8dbe4b709c44366699b0c
+  promql-transform-amd64:
+    type: file
+    description: promql-transform binary for adding label selectors on amd64
+    filename: promql-transform
+  promql-transform-arm64:
+    type: file
+    description: promql-transform binary for adding label selectors on arm64
```

Comment:
Missing a file name here?

Reply:
That was sloppy of me! Thanks for pointing it out. Fixed.